product_packaging_level_pallet/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+{
+    "name": "Product Packaging Type Pallet",
+    "summary": "Manage packaging of pallet type",
```

Comment:
```suggestion
    "summary": "Manage packaging level of pallet",
```

Reply:
No, don't change this, it is a package type specialization, not a package level